src/verifiers/openAttestationDnsTxt.ts

```diff
@@ -103,7 +109,7 @@ export const openAttestationDnsTxt: Verifier<
       }
       const documentData = getData(document);
```

Comment:
either a comment or an explicit `else if` here would be nice because i was confused about where v3 was being handled when i saw v2 was being handled above

Reply:
remove no-else-return eslint rules, added a comment and else block :)

Reply:
or you could do else if (test for v3) { }
else { throw new Error("wtf you ain't supposed to be here") }

Reply:
Nah, the check is just to get correct typings and autocomplete, otherwise we wouldn't do it. in the `else` branch`, type is correct, sad if you did something weird it will probably throw and be caught by the try/catch block